src/main/kotlin/com/liftric/dtcp/tasks/GetOutdatedDependenciesTask.kt

```diff
@@ -51,6 +51,10 @@ abstract class GetOutdatedDependenciesTask : DefaultTask() {
             else -> throw GradleException("Either projectUUID or projectName and projectVersion must be set")
         }
 
+        if (project.directDependencies == null) {
+            throw GradleException("Project does not have direct dependencies")
```

Comment:
Does the project need direct dependencies?

Reply:
It is necessary for the "GetOutdatedDependencies" function. If a project does not have this and a user executes this task, something is probably wrong with his project and he should get a warning.
Otherwise he would always be shown that no dependency has an update available.